### ISSUE TITLE
Static variables are not inspectable in the debugger.

### ISSUE THREAD

**Issue by blubberguts** [godotengine/godot#78170]
### Godot version

 4.1 beta 1

### System information

Godot v4.1.beta1 - Windows 10.0.22621 - Vulkan (Forward+) - dedicated NVIDIA GeForce RTX 3050 Laptop GPU (NVIDIA; 31.0.15.3141) - AMD Ryzen 7 4800H with Radeon Graphics (16 Threads)

### Issue description

At runtime, static variables are not inspectable.

They do not show in the "filter variables" - because they are not considered locals.

If you inspect a class that has static variables, they do not show there either - it lists members and consts, but not statics. It seems the inspector needs a new pane called "statics"

In addition, you cannot hover over the variable in your code and see a popup showing the value - it never pops up, even though other variables do.

And finally they do not show up in code completion either! If you are working in the script that defines them, yes they are visible.
But if you try to use them from a script outside the class definition...they are not visible, BUT THEY STILL WORK!

Ie code completion will not work, but if you ignore it and type it from memory, the code compiles without error and works.

### Steps to reproduce

Include a static variable in a class. 

You will be unable to inspect the value at runtime, and it will not be visible in the inspector for that class.

### Minimal reproduction project

N/A

**Comment by DaveInDev** [comment]
Hi, any news on this issue ? It seems that we still cannot inspect static variables from a breakpoint.

**Comment by blubberguts** [comment]
No, sorry. I actually stopped using Godot a few weeks after this because I felt it was not ready for prime time. I may try it again in a year or two.
    On Monday 9 June 2025 at 04:16:34 am AEST, DaveInDev ***@***.***> wrote:  
 
 DaveInDev left a comment (godotengine/godot#78170)
Hi, any news on this issue ? It seems that we still cannot inspect static variables from a breakpoint.

**Comment by DaveInDev** [comment]
Hi @AThousandShips , can you explain why this was removed from 4.x milestone ?
Does it mean that this problem won't be treated ?
Because it still remains : as far as I tried, I cannot inspect static variables from the debugger...

**Comment by AThousandShips** [comment]
Because the 4.x milestone is not used for bugs, it means nothing for issues and is just misleading, it does not mean it will get any less attention

**Comment by FabioNevesRezende** [comment]
how can the community take so long to fix a bug that is so crucial? I mean, debugging is a basic part in game development...

**Comment by Ctorres01** [comment]
I'm surprised this feature is not already in the engine. I really need this... If someone comes across this message and has the skill to implement this feature, please, do it ! 😃
